README.md

```diff
@@ -81,3 +92,35 @@ If you need to change the transporter use by the application:
  with: 
      TRANSPORTER: nats://nats:4222
```

Comment:
if the port is optional only in this case, can we make it clear that is optional?

Reply:
no, it's not optional